### PR TITLE
Added Arabic Musnad Imam Abu Hanifa

### PR DESCRIPTION
**Pull request by GibreelAbdullah**
Source - https://shamela.ws/book/29120

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -623280,5 +623280,4874 @@
 				}
 			}
 		]
-	}
+	},
+	"abuhanifa":{
+        "metadata":{
+            "name":"Musnad Imam Abu Hanifa",
+            "sections":{
+                "1":"كتاب الإيمان والإسلام والقدر والشفاعة",
```
Comment (fawazahmed0) [inline info.json:623288]: Please use english section names

**Review by fawazahmed0 — CHANGES_REQUESTED** [review]

File: info.json (hunk 1 above)
Comment (GibreelAbdullah) [reply info.json:623288]: English translation is not available.

**Comment by fawazahmed0** [comment]
Jazakallah

**Comment by fawazahmed0** [comment]
Salam, I recently came to know that the book might have issues with authenticity. Please [see this](https://islam.stackexchange.com/a/83172/60349). It might be better to remove the book. Anyways Jazakallah for the hardwork.

**Comment by GibreelAbdullah** [comment]
WaAlaikumAsSalam

This book is famous among the Hanafis as being attributed to Imam Abu Hanifa, even though it is compiled later by the students of his students.

There were various Masaneed of Imam Abu Hanifa and Imam Haskafi combined them into [one book](https://shamela.ws/book/29120)

There are many who have even written Sharh of this book including

- [Mulla Ali Qari d. 1014H](https://shamela.ws/book/9060)
- [Imam Muhammad 'Abid al-Sindi (d. 1257H)](https://archive.org/details/mwahib-latifa-shrah-musnad-abihanifa)

**Comment by fawazahmed0** [comment]
If this book had grades with it, then there would be no issue.
Also if you have noticed, Shuaib Al Arnaut who was a hanafi muhaddith, decided to authenticate musnad ahmad, but not this book.

**Pull request merged**